addons/clones/addon.json

```diff
@@ -41,11 +49,19 @@
       "name": "Show icon only",
       "default": false,
       "type": "boolean"
+    },
+    {
+      "id": "showSpriteCount",
```

Comment:
maybe the setting should be to see one or the other or both? select setting? not really sure tbh...

Reply:
Eh having it as a select setting doesn't really make sense, its not options for the same setting its two different features